galaxy-simple/values.yaml

```diff
@@ -39,6 +39,7 @@ galaxy_node_port_exposed: 30700
 domain: "mydomain.dev" 
 
 # Storage request
+galaxy_create_pvc: true
```

Comment:
@pcm32 Worked ok with default settings - nothing changed. Galaxy up - no error in galaxy-log. Also:  
`shared-volume   195G       RWX           Retain          Bound     default/galaxy-pvc             13m`